client/src/redux/settings/danger-zone-saga.js

```diff
@@ -13,12 +12,16 @@ import { createFlashMessage } from '../../components/Flash/redux';
 
 function* deleteAccountSaga() {
   try {
-    const { data: response } = yield call(postDeleteAccount);
-    yield put(deleteAccountComplete(response));
+    yield call(postDeleteAccount);
+    yield put(
+      createFlashMessage({
+        type: 'info',
+        message: 'Your account has been successfully deleted'
```

Comment:
Can we do the same for reset progress? Send only code 200 and create a message here?

Reply:
Implemented